### ISSUE TITLE
[1.21.4] Mouse buttons are periodically ignored

### ISSUE THREAD

**Issue by esotericdesign** [lvoegl/libanalog-mc#1]
At times, mouse input is ignored as if buttons are stuck. Applies to both LMB and RMB, released and pressed. Works in the main menu already.
Fabric API 0.112.2
Mod version: 0.0.1

**Comment by lvoegl** [comment]
Non-keyboard inputs are now ignored. ea409254910bc0c1b41f7dc9f2cc7d3538419386
